app/Http/Controllers/UsersController.php

```diff
@@ -71,7 +73,11 @@ public function __construct()
 
     public function card($id)
     {
-        $user = $this->lookupUser($id) ?? UserNotFound::instance();
+        try {
+            $user = FindForProfilePage::find($id);
+        } catch (UserProfilePageLookupException $e) {
+            $user = UserNotFound::instance();
+        }
```

Comment:
This is now weird as it goes through `lookupWithHistory`, finds a matching user, and then renders not found if not explicitly requested with json, even if it always returns json.